docs/configuration.md

```diff
@@ -5,27 +5,52 @@ title: Configuration
 * `REACT_APP_USE_PHOTODIODE`![boolean](https://img.shields.io/badge/-boolean-lightgrey)![default false](https://img.shields.io/badge/default-false-yellow)![buildtime](https://img.shields.io/badge/-buildtime-blue): whether the photodiode is in use. This can be used when the task is running as a desktop app (electron).
+
+
+### Desktop and Online
+* `REACT_APP_VOLUME`![boolean](https://img.shields.io/badge/-boolean-lightgrey)![default false](https://img.shields.io/badge/default-false-yellow)![buildtime](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used on both the desktop (electron) and online settings.
 * `REACT_APP_PARTICIPANT_ID` ![string](https://img.shields.io/badge/-string-grey)![optional](https://img.shields.io/badge/-optional-yellow)![runtime](https://img.shields.io/badge/-runtime-purple): The default participant id to show when authorizing a participant. If not set, user will need to enter the value in the provided input box.
 * `REACT_APP_STUDY_ID` ![string](https://img.shields.io/badge/-string-grey)![optional](https://img.shields.io/badge/-optional-yellow)![runtime](https://img.shields.io/badge/-runtime-purple): The default study id to show when authorizing a participant. If not set, user will need to enter the value in the provided input box.
 
-Here are details on some of the badges:
-- ![runtime](https://img.shields.io/badge/-runtime-purple): Run-time environment variable. Set at the system level. See [this tutorial for OS specific instructions.](https://www.imatest.com/docs/editing-system-environment-variables/#Windows)
+### Understanding Build Time and Run Time Variables
+
+- ![runtime](https://img.shields.io/badge/-runtime-purple): Run-time environment variables allow for quick changes before running the task. They are convinient since they don't require building a whole different executable. For a Desktop Application, these are set at the system level and must be available at the time where the task runs. See [this tutorial for OS specific instructions.](https://www.imatest.com/docs/editing-system-environment-variables/#Windows). For a Firebase Application, these must be stored in the Firestore.
+
+- ![buildtime](https://img.shields.io/badge/-buildtime-blue): Build-time environment variable are configured before the application is built and cannot be changed at run-time. They are configured via `.env` files before building. Instead of using a single `.env` file, we prefer to defined separate files under the `env/` directory, and use a package called `dotenv-cli` before building`. 
+
+The `dotenv-cli` commes with the `dotenv` command that can be used to propery load the needed variables. This command does not run directly from the command line (terminal). Instead, to set build-time environment variables, we need to add a  script to `package.json` with the following format:
```

Comment:
is this supposed to be `comes` instead of `commes`

Reply:
Corrected, along with few other typos!